### ISSUE TITLE
[Question] Dalli compatibility?

### ISSUE THREAD

**Issue by aspett** [cjbottaro/cream_ex#2]
Heya @cjbottaro. I see you mention that this package is compatible with Dalli, however upon closer look, it appears to be using a version of Memcachex that does not use flags when writing or reading to/from memcached, meaning it would not be possible to write to using cream_ex, and read with Dalli if Dalli is set up to use the JSON serialiser?

Am I missing something here? Perhaps an override with your fork of memcachex? Cheers

**Comment by cjbottaro** [comment]
Hi, sorry for the late response.

Yeah, when I originally wrote Cream, my intention for "compatibility" was to use the same consistent hashing algorithm Dalli uses that way both can connect to the same memcached cluster. It wasn't until much later that I found out about this flag issue.

It's been a long while, but I _think_ I addressed the issue in this fork of memcachex:
https://github.com/cjbottaro/memcachex/commits/dalli_compatibility

I don't remember how complete that fix is.

Hope that helps somewhat.
